### PR TITLE
Fix Acceptance- Upgrade SO acceptance & fix linting

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
My first aim is to fix the acceptance tests locally, so this may initially fail on Drone

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ Gemfile.lock
 apps/**/translations/en
 coverage
 artefacts
+npm-shrinkwrap.json
```
Comment (JoeChapman) [inline .gitignore]: we should probably remove shrinkwrap

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,5 @@ Gemfile.lock
 apps/**/translations/en
```
Comment (JoeChapman) [inline .gitignore]: ` apps/**/translations/**`

File: .gitignore [hunk 3]
```diff
@@ -7,3 +7,5 @@ Gemfile.lock
...
 coverage
 artefacts
```
Comment (JoeChapman) [inline .gitignore]: Is there a folder called artefacts?

**Comment by lennym** [comment]
I'd strongly prefer to do the rename `acceptance => acceptance_tests` the other way round to standardise on `acceptance`.

Underscores suck.

File: .gitignore (hunk 1 above)
Comment (sulthan-ahmed) [reply .gitignore]: It's removed in this PR

screen shot 2017-11-15 at 11 58 00

File: .gitignore (hunk 3 above)
Comment (sulthan-ahmed) [reply .gitignore]: there's a whole bunch of stuff, I'll remove them

**Comment by sulthan-ahmed** [comment]
good point, I shouldn't have changed it.  I was just following the README here https://github.com/UKHomeOffice/so-acceptance/blame/master/README.md#L28 
assuming that was the best practice and the way to get it working.  Clearly it wasn't

**Comment by lennym** [comment]
I'm not sure there's any value at all in fixing tests for things that we're about to delete.

**Comment by lennym** [comment]
I've deleted basically everything, so you might be better of reviewing from https://github.com/UKHomeOffice/rotm/tree/RTMD-128/fix/acceptance-tests rather than reviewing the diff.

**Pull request merged**